Source/Manager/User/User.swift

```diff
@@ -292,6 +292,38 @@ public class User: UserProtocol {
         isPersistent = true
     }
 
+    private static let keyPrefix = "old-sdk-app-transfer-"
+
+    func storeOnDevice(key: String) throws {
```

Comment:
Should use `rethrows`

Reply:
Why rethrows though?
_"A function or method can be declared with the rethrows keyword to indicate that it throws an error only if one of its function parameters throws an error"_
https://docs.swift.org/swift-book/ReferenceManual/Declarations.html